src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -310,9 +310,22 @@ export class RunParametersAction extends TooltipFromLabelAction {
 		let defaultParameters = new Map<string, string>();
 		editor.cells.forEach(cell => {
 			if (cell.isParameter) {
+				if (!cell.source) {
+					// If there is no parameters in the cell indicate to user to add them
+					this.notificationService.notify({
+						severity: Severity.Info,
+						message: localize('noParametersInCell', "This notebook cannot run with parameters until a there are parameters added to the parameter cell. [Learn more](https://docs.microsoft.com/sql/azure-data-studio/notebooks/notebooks-parameterization)."),
```

Comment:
Is this really true?  I would expect that we could still run - it just would run the Notebook as normal since there's no parameters to use.

Reply:
Yes, we should not run the normal notebook with no parameters as that would not be useful for parameterization. As such I think it would be better to tell the user to learn more about parameterization via the link and therefore help them understand that the parameter cell is for parameters.